app/assets/stylesheets/activeadmin_addons/all.scss

```diff
@@ -34,3 +36,20 @@ input[type=text] {
     width: 110px;
   }
 }
+
+.filter_range_date_time_picker {
+  input[type=text] {
+    padding-right: 0;
```

Comment:
@ldlsegovia 

Based on stylelint I shouldn't use `input[~]` but I copied this from the code for range filter. Should I change that one too? 

I googled the error to know what to solve: https://stylelint.io/user-guide/rules/selector-no-qualifying-type/